src/Models/MiningReward.ts

```diff
@@ -14,7 +14,7 @@ export class MiningReward extends Model {
                 allowNull: false,
             },
             rewardAmount: {
-                type: DataTypes.STRING,
+                type: DataTypes.DECIMAL(32),
```

Comment:
NUMERIC or DECIMAL? Does it make any difference?

Reply:
Sequelize only knows about the decimal data type so it's actually transformed into numeric which is set up in the migration file.

Reply:
I think they are the same in behavior but both supported by postgresql so that's weird. Okay